modules/homa-lite/src/lib.rs

```diff
@@ -249,7 +255,7 @@ pub mod module {
 	/// xcm_dest_weight: value: Weight
 	#[pallet::storage]
 	#[pallet::getter(fn xcm_dest_weight)]
-	pub type XcmDestWeight<T: Config> = StorageValue<_, Weight, ValueQuery>;
+	pub type XcmDestWeight<T: Config> = StorageValue<_, XcmWeight, ValueQuery>;
```

Comment:
The origin `Weight` is `pub type Weight = u64;`
https://github.com/paritytech/substrate/blob/ca1f81cd0864ae0614aaff07fa121d2d42110586/frame/support/src/weights.rs#L150

OldWeight : `pub struct OldWeight(pub u64);` 
https://github.com/paritytech/substrate/blob/0dc5358e43187a1f54afb2455d42eddb0965c16a/primitives/weights/src/lib.rs#L77

The new `Weight` is 
```
pub struct Weight {
	#[codec(compact)]
	/// The weight of computational time used based on some reference hardware.
	ref_time: u64,
	#[codec(compact)]
	/// The weight of storage space used by proof of validity.
	proof_size: u64,
}
```
https://github.com/paritytech/substrate/blob/0dc5358e43187a1f54afb2455d42eddb0965c16a/primitives/weights/src/weight_v2.rs#L29-L36

XcmWeight: `pub type Weight = u64;`
https://github.com/paritytech/polkadot/blob/f85f96c9d6f4d16bc04f2b3950b091c979adf6d0/xcm/src/v2/mod.rs#L222

use `XcmWeight` instead of the origin `Weight` is the same.